Source/Fuse.Drawing.Surface/CoreGraphics/NativeSurface.uno

```diff
@@ -0,0 +1,48 @@
+using Uno;
+using Uno.Collections;
+using Uno.Compiler.ExportTargetInterop;
+using Fuse.Drawing.Primitives;
+
+namespace Fuse.Drawing
+{
+	extern(iOS||OSX)
+	class NativeSurface : CoreGraphicsSurface
+	{
+
+		IntPtr _cgContext = IntPtr.Zero;
+
+		public void Begin(IntPtr cgContext, float pixelsPerPoint)
+		{
+			// The CGContext is already scaled
+			_pixelsPerPoint = 1.0f;//pixelsPerPoint;
```

Comment:
Fix in https://github.com/fusetools/fuselibs-public/pull/244/commits/db27316bd8ad9a759fe843f0ae8bb0cc42ff6227